contents/blog/inflated-risk-seems-riskier.md

```diff
@@ -0,0 +1,78 @@
+
+What we realized was we wanted to do something bigger, and we were happy not to own the whole pie. Tim and I had been pushing for months and stopped to ask ourselves, if we continued on this path and we reached 80 years old, would we be happy with what we did in our career? We both said no, for us it was slow and dull. We decided to aim bigger and do something more exciting.
+
+We reorganized everything around what were often weekly product launches. Yes that’s right, not sprints, build and validate the product with 10-15 customers in a week. It sounds ludicrous, and it was. But demonstrating and giving access to a product rather than a landing page or mockups got us the high quality validation we needed to move on to the next idea. We got comfortable throwing out what we’d built - as cliché as it sounds, the light bulb moment is realizing you never throw away the lessons. If you’d like more detail, I wrote more on [pivoting over and over](/story-about-pivots).
+
+<iframe src="https://giphy.com/embed/UE9JPTe7LySli" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/phone-working-busy-UE9JPTe7LySli">via GIPHY</a></p>
+
+## Don't buy all the things
+
+As conventional wisdom goes, we spent very little money in the early days. GitHub, AWS and Figma free tier were our bread and butter but it took longer to discover the true value of your time and how to spend it wisely.
+
+Tim and I tried remote working, but found it difficult to be focused and in-sync with each other while rapidly prototyping. Getting cheap WeWork access ($100/month/person) was a game changer for us. We constantly pushed each other and tried to keep up with the other’s pace - plus all the free coffee we could drink!
+
+<iframe src="https://giphy.com/embed/oGP0Sv692lb68" width="480" height="269" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/coffee-pot-problem-oGP0Sv692lb68">via GIPHY</a></p>
+
+While Tim was building products, I recruited users to validate them. It was manual and slow so I worked out the value of my time (based on the opportunity cost of not having a normal job), and realized that a little ad spend could get weeks' worth of manually-recruited users in a day. If you are in a similar situation, ask yourself if there are legitimate reasons to go slowly or are you artificially holding yourself back?
+
+## Closing our first round
+
+Both Tim and I wanted to get into YC from the moment we decided to go for the VC route. We reached out to our network and scoured the internet to gleam application strategies.
```

Comment:
Could link to the part of the fundraising blog post that talks about the ways to raise